src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax;
+
+public class MainApp {
+    private static final String EMPTY_STRING = "";
+    private static final String SAVE_TEXT_TO_FILE_QUESTION =
+            "Do you want to save this text to file?";
+    private static final String REQUEST_FILE_PATH_TEXT = "Input file path: ";
+    private static final String REQUEST_FILE_NAME_TEXT = "Input file name: ";
+    private static final String WRONG_NUMBER_OF_ARGS_ERROR_MSG =
+            "Wrong number of arguments for %s command.\n";
+    private static final String REG_EXP_MULTI_WHITESPACE = "[\\s]+";
+    private static final String CREATE_COMMAND = "CREATE";
+    private static final String READ_COMMAND = "READ";
+    private static final String INFO_COMMAND = "INFO";
+    private static final String HELP_COMMAND = "HELP";
+    private static final String EXIT_COMMAND = "EXIT";
+
+    public static void main(String[] args) {
+        for (; ; ) {
```

Comment:
move all logic, constants  to separate class.And call it's methods from main  method